src/providers/google-translate.provider.js

```diff
@@ -26,7 +26,7 @@ class GoogleTranslateProvider {
         to,
       });
 
-      return translation.text;
+      return translation[0];
```

Comment:
```javascript
const [translation] = await translate...
...
return translation;
```

Reply:
Boa! Vou testar aqui

Reply:
@rodrigogs , alguma recomendação de testes unitários ou de integração pra essa classe?

Reply:
> ```js
> const [translation] = await translate...
> ...
> return translation;
> ```

Funcionou!

Reply:
It would be grat to at least mock the Google Translate API(I recomend to use jest.fn() to mock), and test it against the translation provider. It should be a nice unit test for this feature.

A integration test could use almost the same code without mocking the Google Translate API. It should be great to test if translate service is still working and using the same contract.

Stay aware of the glossary provider, it should be considered and testes too.

Reply:
All right! Thankss